handlers/watchers/watchers.ini

```diff
@@ -1,12 +1,13 @@
 [servo/servo]
-emilio = components/script/dom/webgl components/canvas/webgl ports/geckolib components/style/ components/layout/
-bholley = components/style/
-Wafflespeanut = python/
-jgraham = components/webdriver_server/ tests/wpt/ -tests/wpt/mozilla -tests/wpt/metadata -tests/wpt/web-platform-tests -tests/wpt/metadata-css -tests/wpt/css-tests
 KiChjang = components/script components/script_traits components/net components/net_traits
+Wafflespeanut = python/
```

Comment:
Shouldn't make a difference technically (Github usernames and labels are case-insensitive, and highfive too), but for consistency, you bet :)